code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -34,6 +34,7 @@
 
 	var/disabling_timer_id = null
 	var/list/player_access = list()
+	/// 1 = emag unlocked, 2 = open panel and emagged
 	var/emagged = 0
```

Comment:
While you're here, could you change this to use a define rather than just magic numbers?

Reply:
I am already planning on doing a follow up PR. I dont want to hold this PR further, as it also has some bug fixes baked in it.

I will do it in the follow up PR. Lmk if thats ok

Reply:
I think I will just turn this into emagged true/false. discussed on discord.

Reply:
Can you check this?
b02a36e